libtcmu_common.h

```diff
@@ -53,6 +53,8 @@ struct tcmulib_cmd;
 
 #define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
 
+#define VPD_MAX_WRITE_SAME_LENGTH 0xFFFF
```

Comment:
Where did this limit come from? Is it a vmware limit or default? It seems like we have 2 more bytes in the number of logical blocks field for write same 16.